test/e2e/stop_test.go

```diff
@@ -181,6 +181,18 @@ var _ = Describe("Podman stop", func() {
 		Expect(strings.TrimSpace(finalCtrs.OutputToString())).To(Equal(""))
 	})
 
+	It("podman stop container --timeout Warning", func() {
+		SkipIfRemote("warning will happen only on server side")
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "test5", ALPINE, "sleep", "100"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		session = podmanTest.Podman([]string{"stop", "--timeout", "1", "test5"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		warning := session.ErrorToString()
+		Expect(warning).To(ContainSubstring("Timed out stopping container"))
```

Comment:
I think you mean `StopSignal TERM failed to stop container` here (the `Timed out` one is now just debug).

But I still don't understand @mheon's reason for requesting two messages.

Reply:
We don't want to print the error message every time, it's largely irrelevant to the situation and will reduce clarity (I expect it's going to be rather long - "failed to stop PID %d after %d seconds" or somesuch.

What we actually want is to print the "Timed out stopping container %s" message first as a logrus.Warnf, then print "Error stopping container %s using stop signal: %v" including the error message as a Debugf so the error isn't entirely lost.